boards/arm/adafruit_feather_nrf52840/doc/index.rst

```diff
@@ -103,33 +132,72 @@ Push buttons
 Programming and Debugging
 *************************
 
-Applications for the ``adafruit_feather_nrf52840`` board configuration
-can be built and flashed in the usual way (see :ref:`build_an_application`
-and :ref:`application_run` for more details).
+The Feather nRF52840 ships with the `Adafruit nRF52 Bootloader`_ which
+is compatible with BOSSA and supports flashing using `UF2`_. This allows
+easy flashing of new images, but does not support debugging the device.
+
+Additionally, if :code:`CONFIG_USB_CDC_ACM` is enabled then the BOSSA
+bootloader will be entered automatically when you run :code:`west flash`.
```

Comment:
I don't have one of these boards to test but is this really true?

Reply:
This is again copied from another board without a lot of understanding, in this case the [Adafruit ItsyBitsy M4 Express](https://github.com/bkietz/zephyr/blob/5bc208e16c7ae6f94e56efa5fd5be6829b8dc35e/boards/arm/adafruit_itsybitsy_m4_express/doc/index.rst?plain=1#L112-L120)

Reply:
I *can* verify that with these options `west flash` invokes BOSSA

Reply:
Invokes sure, but if you have a board running a zephyr application and run west flash, it somehow reboots it into bootloader mode?

Reply:
I haven't managed a zephyr application yet, but it does reboot it into bootloader mode from a running arduino application